src/extension.ts

```diff
@@ -62,11 +62,14 @@ export async function activate(context: vscode.ExtensionContext) {
     juliaexepath.activate(context, g_settings);
     repl.activate(context, g_settings);
     weave.activate(context, g_settings);
-    tasks.activate(context, g_settings);
     smallcommands.activate(context, g_settings);
     packagepath.activate(context, g_settings);
     openpackagedirectory.activate(context, g_settings);
 
+    if (vscode.workspace.rootPath) {
+        tasks.activate(context, g_settings);
+    }
+

```

Comment:
This seems unrelated? If this fixes a bug, can you please move it into a separate PR?